.lavra/memory/recall.sh

```
@@ -0,0 +1,152 @@
+    grep "\"bead\":\"$CHILD_ID\"" "$KNOWLEDGE_FILE" 2>/dev/null
+  done | jq -r '"\(.type | ascii_upcase): \(.content)"' 2>/dev/null
+  exit 0
+fi
+
+# Build input (optionally include archive)
+INPUT_FILES="$KNOWLEDGE_FILE"
+$INCLUDE_ARCHIVE && [[ -f "$ARCHIVE_FILE" ]] && INPUT_FILES="$ARCHIVE_FILE $KNOWLEDGE_FILE"
+
+# Recent mode
+if [[ "$RECENT" -gt 0 ]]; then
+  cat $INPUT_FILES | tail -"$RECENT" | jq -r '"\(.type | ascii_upcase): \(.content)"' 2>/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)